examples/check_integration_error.py

```diff
@@ -6,123 +6,135 @@
+clean_data.reorder(["vd", "t2"], first=False)
 clean_data["t2":0] *= 0.5
 fake_data_noise_std = 2.0
 clean_data.reorder(["ph1", "ph2", "vd"])
-bounds = (0, 200)  # seem reasonable to me
-result = 0
-n_repeats = 100
-all_results = ndshape(clean_data) + (n_repeats, "repeats")
+# Unitary FT concentrates signal into a single index/position along coherence
+# transfer pathway while preserve the vector norm so divide by sqrt(N_ph)
+clean_data /= np.sqrt(
+    psd.ndshape(clean_data)["ph1"] * psd.ndshape(clean_data)["ph2"]
+)
+# }}}
+# Allocate nddata to place the calculated integrals with error into
+all_results = psd.ndshape(clean_data) + (n_repeats, "repeats")
 all_results.pop("t2").pop("ph1").pop("ph2")
-all_results = all_results.alloc()
-all_results.setaxis("vd", clean_data.getaxis("vd"))
-print("shape of all results", ndshape(all_results))
-for j in range(n_repeats):
-    data = clean_data.C
-    data.add_noise(fake_data_noise_std)
-    # at this point, the fake data has been generated
-    data.ft(["ph1", "ph2"])
-    # {{{ usually, we don't use a unitary FT -- this makes it unitary
-    data /= 0.5 * 0.25  # the dt in the integral for both dims
-    data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
-    # }}}
-    dt = diff(data.getaxis("t2")[r_[0, 1]]).item()
-    data.ft("t2", shift=True)
-    # {{{
-    data /= sqrt(ndshape(data)["t2"]) * dt
-    error_pathway = (
-        set(
-            (
-                (j, k)
-                for j in range(ndshape(data)["ph1"])
-                for k in range(ndshape(data)["ph2"])
+all_results = all_results.alloc().setaxis("vd", clean_data.getaxis("vd"))
+with psd.figlist_var() as fl:
+    for j in range(n_repeats):
+        data = clean_data.C  # need a copy since we are looping
+        data.add_noise(fake_data_noise_std)  # add noise to each individual
+        #                                     repeat
+        # at this point, the fake data has been generated
+        data.ft("t2")
+        dt = data.get_ft_prop("t2", "dt")
+        # {{{ Vector-normalize the FT
+        data /= np.sqrt(psd.ndshape(data)["t2"]) * dt
+        # }}}
+        if j == 0:
+            s_int, frq_slice = integral_w_errors(
+                data,
+                signal_pathway,
+                fl=fl,
+                return_frq_slice=True,
+            )
+        else:
+            s_int, frq_slice = integral_w_errors(
+                data,
+                signal_pathway,
+                fl=None,
+                return_frq_slice=True,
             )
-        )
-        - set(excluded_pathways)
-        - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
+        manual_bounds = select_pathway(data["t2":frq_slice], signal_pathway)
+        N = psd.ndshape(manual_bounds)["t2"]
+        df = manual_bounds.get_ft_prop("t2", "df")
+        manual_bounds.integrate("t2")
+        # N terms that have variance given by fake_data_noise_std**2 each
+        # multiplied by df
```

Comment:
this was a comment carried over - if you think unnecessary or doesn't apply I can easily delete